ribs.py

```diff
@@ -123,18 +141,30 @@ def main():
 
     frame_clock = pg.time.Clock()
 
+    time = 0
+    delta_time = 1 / 60
+    num_teapots = 1
     while is_running():
         # Tell Pygame we're on a new frame, with the given framerate
         # set it to zero to unlimit.
         frame_clock.tick(60)
+        time += delta_time
         # See what buttons are pressed this frame.
         process_events()
 
         if key_pressed("A"):
-            print("Pressed the button")
+            num_teapots += 1
+
+        for i in range(num_teapots):
+            import math
+            r = 100
+            x = math.cos(i * 10 / 50 + time) * r + SCREEN_WIDTH / 2
+            y = math.sin(i * 10 / 50 + time) * r + SCREEN_WIDTH / 2
```

Comment:
Spooky math :ghost: 

A comment about what it does might be nice

 (also 10/50 == 1/5)

Reply:
Not sure how long this code will be there. And how high should the comment to code factor be? Is it really that much easier for them to read heavily commented code?I think there's a sweetspot, and I'm kinda unsure where it is.

I honestly want to remove half of these comments, since they're completely pointless. Given the point you made about not teaching them bad habits, I'd say over commenting your code is one of them.

Reply:
See other comment

In this case, I think there should be a high level description like "make the image move back and forth along the screen."